test/src/org/omegat/languagetools/LanguageToolTest.java

```diff
@@ -99,14 +99,10 @@ public void testRemoteServer() throws Exception {
         try {
             server.run();
 
-            try {
-                new LanguageToolNetworkBridge(SOURCE_LANG, TARGET_LANG, "http://localhost:8081");
-                fail("URL not specifying API v2 should fail due to XML response instead of JSON");
-                // TODO: LanguageTool will drop XML entirely in version 3.6; this
-                // test might need to be adjusted then.
-            } catch (Exception e) {
-                // OK
-            }
+            assertThrows("URL not specifying API actions should fail due to missing argument.",
+                    java.lang.Exception.class,
```

Comment:
Is it really good when LanguageToolNetworkBridge class throws generic `Exception` class other than specific exception that has a reason?

Reply:
I will raise another refactoring PR for it.